mmedit/models/video_interpolators/basic_interpolator.py

```diff
@@ -268,17 +269,18 @@ def split_frames(input_tensors):
         frames.
 
         Args:
-            input_tensors (Tensor): Tensor of input frames.
+            input_tensors (Tensor): Tensor of input frames with shape
+                [1, t, c, h, w]
 
         Returns:
-            list[Tensor]: A list of split tensors.
+            Tensor: Split tensor with shape [t-1, 2, c, h, w]
         """
 
-        num_frames = input_tensors.shape[0]
+        num_frames = input_tensors.shape[1]
 
-        result = []
-        for i in range(0, num_frames - 1, 1):
-            result.append(input_tensors[i:i + 2])
+        result = input_tensors[:, :2]  # the first tensor
+        for i in range(1, num_frames - 1, 1):
+            result = torch.cat([result, input_tensors[:, i:i + 2]], dim=0)
```

Comment:
Here `torch.cat` is called multiple times. I suggest we do the following:
```
result = [input_tensors[:, i:i + 2] for i in range(0, num_frames - 1)]
result = torch.cat(result, dim=0)
```

Reply:
Good idea, thank you